docs/guides/color.md

```diff
@@ -0,0 +1,290 @@
+function is responsible for coercing color input values to the standard form and supports a variety of inputs.
+
+```{note}
+The [`ColorType`](https://github.com/napari/napari/blob/43d9b59da89b9a2b97dba3967a7b28526f9aa0ce/napari/utils/colormaps/colormap_utils.py#L17)
+type definition almost describes all the types of acceptable color inputs, except that it includes
+vispy's `Color` and `ColorArray`, which are not accepted by `transform_color`.
+```
+
+
+### Supported single color input
+
+The single face color could have been defined as red in many equivalent ways.
+
+```python
+# CSS3 name
+points.face_color = 'red'
+
+# matplotlib single character name
+points.face_color = 'r'
+
+# RGB or RGBA hex-code
+points.face_color = '#ff0000'
+points.face_color = '#ff0000ff'
+
+# RGB or RGBA tuple, list, or array
+points.face_color = (1, 0, 0)
+points.face_color = (1, 0, 0, 1)
+points.face_color = [1, 0, 0]
+points.face_color = [1, 0, 0, 1]
+points.face_color = np.array([1, 0, 0])
+points.face_color = np.array([1, 0, 0, 1])
+```
+
+A named color string must either be from the [CSS3 specification](https://www.w3.org/TR/css-color-3/#svg-color)
+or must be one of [matplotlib's single character shorthand names](https://matplotlib.org/stable/tutorials/colors/colors.html#specifying-colors).
+
+
+### Supported multiple color input
+
+Sometimes there is a need to specify many colors at once.
+For example, the face colors of all the points in a layer can be specified individually
+
+```python
+In [3]: points.face_color = ['red', 'lime', 'blue']
+```
+
+which will then be coerced into the standard array form.
+
+```python
+In [4]: points.face_color
+Out[4]:
+array([[1., 0., 0., 1.],
+       [0., 1., 0., 1.],
+       [0., 0., 1., 1.]])
+```
+
+For the most part, any tuple, list, array, or generator of single colors
+is acceptable input when specifying multiple colors.
+
+```python
+# Tuple, list, array, or generator of names
+points.face_color = ('red', 'lime', 'blue')
+points.face_color = ['red', 'lime', 'blue']
+points.face_color = np.array(['red', 'lime', 'blue'])
+points.face_color = c for c in ('red', 'lime', 'blue')
+
+# List of RGB or RGBA tuples or lists
+points.face_color = [
+    (1, 0, 0),
+    (0, 1, 0),
+    (0, 0, 1),
+]
+points.face_color = [
+    [1, 0, 0, 1],
+    [0, 1, 0, 1],
+    [0, 0, 1, 1],
+]
+
+# (N, 3) RGB or (N, 4) RGBA array
+points.face_color = np.array([
+    [1, 0, 0],
+    [0, 1, 0],
+    [0, 0, 1],
+])
+points.face_color = np.array([
+    [1, 0, 0, 1],
+    [0, 1, 0, 1],
+    [0, 0, 1, 1],
+])
+```
+
+However, when specifying many colors in a sequence or generator *do not mix different single color types*.
+For example, something like
+
+```python
+In [5]: points.face_color = np.array(['red', (0, 1, 0), '#0000ff'])
+```
+
+is not supported or guaranteed to work.
+
+
+## Colormaps
+
+In general, a colormap is a function that maps from an arbitrary domain to a some color space range.
```

Comment:
This feels a little jargon-ey to me. It might be fine, as this is a document is targeted towards. Maybe something like this?

```suggestion
In general, a colormap is a function that specifies how data values are represented as colors.
```

Reply:
You're right. I went for something slightly different and shorter.

```A colormap maps data values to corresponding RGBA colors.```

Then modified the text around that appropriately. Please take another look in the next push.

Reply:
That's a great sentence.

Reply:
> Then modified the text around that appropriately. Please take another look in the next push.

Looks good to me!